app/models/place.rb

```diff
@@ -0,0 +1,4 @@
+class Place < ApplicationRecord
+  belongs_to :district
+   enum status: {hotel: 0, restaurent: 1}
```

Comment:
```suggestion
  enum status: {hotel: 0, restaurent: 1}
```
xóa space ở đầu đi em

Reply:
:(((((((((((